server/src/rsvp/rsvp.controller.ts

```diff
@@ -147,10 +164,15 @@ export class RsvpController {
 					);
 				}
 			}
-			rsvpInfo.associates = associates;
 		}
 
-		return { ...guest, ...contactInfo, ...rsvpInfo };
+		return {
+			primary: !!guest,
```

Comment:
:/ why

Reply:
idk, not sure what to do with the return yet but I can send back everything or nothing if you want

Reply:
but why are we doing booleans? I literally don’t understand sorry

Reply:
Returning everything again